### PR TITLE
Added Dockerfile and build script

### PR DESCRIPTION
**Pull request by RoastVeg**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM i386/alpine
+
+RUN apk update
+RUN apk add --no-cache build-base libpng-dev wine ncurses freetype
+RUN apk add binutils-arm-none-eabi --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted
+
+ENV BUSYBOX 1
+WORKDIR /app
+
+CMD ["/bin/sh"]
```
Comment (PikalaxALT) [inline Dockerfile:10]: Missing newline at EOF (github complains)

**Comment by martmists** [comment]
Call me a moron but why exactly do we need a Dockerfile for this?

**Comment by RoastVeg** [comment]
We don't "need" one.

It does make resolving a number of dependencies much easier, including wine and arm cross compilers, doesn't harm the rest of the codebase, and provides an alternative route into getting started with the project.

I personally prefer to work this way because it keeps my host machine very tidy, but YMMV.

**Comment by RoastVeg** [comment]
Ready after #182

**Pull request merged**